pkg/controller/endpointslice/endpointset.go

```diff
@@ -68,7 +68,7 @@ func (s endpointSet) Has(item *discovery.Endpoint) bool {
 
 // Returns an endpoint matching the hash if contained in the set.
 func (s endpointSet) Get(item *discovery.Endpoint) *discovery.Endpoint {
-	got, _ := s[hashEndpoint(item)]
+	got := s[hashEndpoint(item)]
```

Comment:
return s[hashEndpoint(item)]

Reply:
Thanks for your suggestion.
I changed it.